py-polars/tests/unit/test_datelike.py

```diff
@@ -551,13 +550,16 @@ def test_date_range_lazy() -> None:
         ).date.tolist()
     )
 
-    # lazy date range with expressions
+
+@pytest.mark.parametrize("low", ["start", pl.col("start")])
+@pytest.mark.parametrize("high", ["stop", pl.col("stop")])
+def test_date_range_lazy_with_expressions(low: str | pl.Expr, high: pl.Expr) -> None:
```

Comment:
The `high` param is missing the `str` option. Not sure why `mypy` isn't complaining about this 🤔 

```suggestion
def test_date_range_lazy_with_expressions(low: str | pl.Expr, high: str | pl.Expr) -> None:
```

Reply:
right, thanks for catching this!